Mlem/App/Views/Shared/Toast/ToastModel.swift

```diff
@@ -13,28 +13,28 @@ class ToastModel {
     
     static let main: ToastModel = .init()
     
-    func activeToast(location: ToastLocation) -> Toast? {
-        toasts.first(where: { $0.location == location })
+    func activeToasts(location: ToastLocation) -> [Toast] {
+        Array(toasts.lazy.filter { $0.location == location }.prefix(3))
     }
     
-    func add(_ type: ToastType, location: ToastLocation? = nil, group: String? = nil) {
+    func add(_ type: ToastType, location: ToastLocation? = nil, important: Bool? = nil) {
+        let important = important ?? type.important
         let newToast: Toast = .init(
             type: type,
             location: location ?? type.location,
-            group: group
+            important: important
         )
```

Comment:
For consistency with suggestion below (and to save a line), this can be

```
let newToast: Toast = .init(
    type: type,
    location: location ?? type.location,
    important: important ?? type.important
)
```

And then use `newToast.important` instead of `important` below